web/lib/supabase/groups.ts

```diff
@@ -13,43 +13,54 @@ export type SearchGroupInfo = Pick<
 >
 
 export async function searchGroups(prompt: string, limit: number) {
-  const query = db
-    .from('groups')
-    .select(
-      'id, data->name, data->about, data->slug, data->totalMembers, data->totalContracts, data->anyoneCanJoin'
-    )
+  const query = selectFrom(
+    db,
+    'groups',
+    'id',
```

Comment:
I think James might prefer the property names in an object as the third parameter rather than just a list of parameters.

Reply:
Hmm, I see your guys' point. I think I will change it. That will also make it doable to select both inside-of JSON fields and other fields in the same query and keep everything straight and correctly typed.